be/src/olap/base_compaction.cpp

```diff
@@ -87,6 +93,11 @@ OLAPStatus BaseCompaction::pick_rowsets_to_compact() {
     }
 
     double base_cumulative_delta_ratio = config::base_cumulative_delta_ratio;
+    if (base_size == 0) {
```

Comment:
We have checked `input_rowsets.size ` firstly, so _input_rowsets.size() at least should be 2 and the base_size should larger than zero.
```
   if (_input_rowsets.size() <= 1) {
        return OLAP_ERR_BE_NO_SUITABLE_VERSION;
    }
```

Reply:
`_input_rowsets.size()` is the number of rowsets.
For example, we have 2 rowsets with version: [0-1] and [2-2];
the `base_size` here is the data size of [0-1], which can be zero.

Reply:
OK.  I see.